src/components/ProductItem.jsx

```diff
@@ -64,7 +57,7 @@ const ProductItem = ({product}) => {
                         ?
                         <div onClick={()=>navigate("/cart")}><span className="add-to-bag material-icons md-24">shopping_bag</span></div>
                         :
-                        <div onClick={()=>addToCart(product)}><span className="add-to-bag material-icons md-24">add</span></div>
+                        <div onClick={()=>handleCart(product)}><span className="add-to-bag material-icons md-24">add</span></div>
                     }
```

Comment:
you can extract this as a component outside of the UI with a name like `CartButton` or something else

Reply:
Correct. Will do this to. Thanks for the idea.